artifactory/commands/transferfiles/transfer.go

```diff
@@ -380,6 +382,24 @@ func (tdc *TransferFilesCommand) createTransferDir() error {
 	return errorutils.CheckError(os.MkdirAll(transferDir, 0777))
 }
 
+// Assert the transfer dir is not in the old structure with a united state.json for all repository.
+// There is no means of conversion to the new structure, where every the state is repository specific and separated.
```

Comment:
There is no means... --> There are no means...
...where every the state is repository specific and separated. --> [probably needs some rephrasing]